client/beadledom-client-example/example-service/src/main/java/com/cerner/beadledom/client/example/FauxModule.java

```diff
@@ -46,8 +46,7 @@ protected void configure() {
   ObjectMapper provideObjectMapper() {
     ObjectMapper objectMapper = new ObjectMapper();
 
-    objectMapper.setPropertyNamingStrategy(
-        PropertyNamingStrategy.CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+    objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
```

Comment:
I think this should still be the old camel case strategy.

Reply:
Changed it back: https://github.com/cerner/beadledom/pull/133/commits/c57a76907e4a578cee170943dadbd96791f92dcc